README.md

```diff
@@ -85,13 +89,7 @@ These commands will update the configuration key (see also [settings documentati
 
 ## Documentation
 
-### Setup documentation
-
-The documentation that covers configurations and design philosophy can be found in the [docs](docs/) directory of this repository. You are recommended to take a look at a documentation of respective version of OmiseGO eWallet Server you are running.
-
--   [latest](https://github.com/omisego/ewallet/tree/master/docs)
--   [v1.1](https://github.com/omisego/ewallet/tree/v1.1/docs)
--   [v1.0](https://github.com/omisego/ewallet/tree/v1.0/docs)
+All documentations can found in the [docs](docs/) directory. You are recommended to take a look at the documentation of respective version of OmiseGO eWallet Server you are running.
```

Comment:
... It is recommended to take a look at the documentation of the eWallet Server version you are running.